### PR TITLE
External location for puppetwrapper configuration property

### PR DESCRIPTION
**Pull request by jmmovilla**
******* REVIEWERS ****
@jframos 

Testbed in testbed

### CODE REVIEW

File: src/main/filters/development.properties [hunk 1]
```diff
@@ -13,7 +13,7 @@ mongo.port=27017
 keystoneURL=http://130.206.80.57:4731/v2.0/
 adminUser=admin
 adminPass=8fa3c69e4c3e9fafa61
```
Comment (flopezag) [inline src/main/filters/development.properties]: This information should not appear here, you should leave it in blank. and define in the readme that you need to update these attributes with the proper values.

File: src/main/filters/mac.properties [hunk 2]
```diff
@@ -10,10 +10,10 @@ mongo.host=127.0.0.1
 mongo.port=27017
 
 #others
-keystoneURL=http://130.206.80.57:4731/v2.0/
+keystoneURL=http://cloud.lab.fi-ware.org:4731/v2.0
 adminUser=admin
 adminPass=8fa3c69e4c3e9fafa61
```
Comment (flopezag) [inline src/main/filters/mac.properties]: Idem

**Comment by jframos** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**